### DISCUSSION TITLE
All /admin/ subpaths give 404

### DISCUSSION THREAD

**Discussion by Joly0** [dani-garcia/vaultwarden#4350] [Q&A]
### Subject of the issue

Hey guys, i tried to add a new user to my vaultwarden instance, but now all subpaths below /admin give me an 404 error (but a bitwarden one, not a normal browser one). I can access /admin normally, but all other pages like "users" or "organization" dont work.

I think this might be cause of reverse proxy settings, because when i try to access those pages through ip rather than domain, i can access all subpaths just fine, but i am not sure, where my issue is. Also it was working about 1-2 months ago without an issue.

For reverse proxy i am using nginx proxy manager and the /admin config there looks like this:
```
location /admin {
    set $forward_path "";
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection $connection_upgrade;
    proxy_set_header X-Forwarded-Host $host/admin;
    if ($forward_path = "") {
        rewrite ^/admin(/.*)$ $1 break;
    }
    proxy_pass http://vaultwarden:80;

    allow 10.50.0.0/24;
    deny all;
}
```
### Deployment environment

* vaultwarden version:

* Install method: Docker

* Clients used: 

* Reverse proxy and version: Nginx Proxy Manager (latest)

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

**Comment by BlackDex** [comment]
Double check the wiki proxy examples.
Not sure what you are trying to do with the path checking there.
It also could be the ip check you do there. If somehow that is not the IP Nginx sees.
